src/theme/DocItem/Content/index.tsx

```diff
@@ -19,24 +20,26 @@ import { useLocation } from "@docusaurus/router";
  - the markdown content does not already contain a top-level h1 heading
 */
 function useSyntheticTitle(): string | null {
-  const {metadata, frontMatter, contentTitle} = useDoc();
+  const { metadata, frontMatter, contentTitle } = useDoc();
   const shouldRender =
-    !frontMatter.hide_title && typeof contentTitle === 'undefined';
+    !frontMatter.hide_title && typeof contentTitle === "undefined";
```

Comment:
Are we running prettier on this codebase? These look like arbitrary style changes, which shoudln't be something we have to spend time discussing as long as we use a formatter.